java/.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "java",
+            "name": "Debug Uyuni",
+            "request": "attach",
+            "hostName": "server.tf.local",
```

Comment:
Thing is that this line will make my config file permanently modified. I do not use local sumaform, but on remote server. I am not sure here, but is there any support for variables in vscode?

Reply:
Thanks, good point, I am trying to find a solution using variables so users can override the default.

Reply:
I couldn't find a good solution to extract only the `hostNames` to another config file (this is already a config file), so the best idea that I have here is to maybe just provide a template file that needs to be copied before usage? `java/.vscode/launch.json` could be added to `.gitignore` then.

Reply:
Changed `launch.json` to a template file (`launch.json.template`), which needs to be copied to `launch.json`.